components/compliance-service/reporting/relaxting/reports.go

```diff
@@ -553,8 +553,14 @@ func convertControl(profileControlsMap map[string]*reportingapi.Control, reportC
 
 	convertedControl.StringTags = jsonTags
 	var jsonRefs []*reportingapi.Ref
-	refs, _ := json.Marshal(profileControl.Refs)
-	json.Unmarshal(refs, &jsonRefs) // nolint: errcheck
+	refs, err := json.Marshal(reportControlMin.Refs)
```

Comment:
[question] Would it help readability if we made this a function with an explicit name?  It is a bit strange since `jsonRefs` is type `[]*reportingapi.Ref`  and `reportControlMin.Refs` is _also_ of type `[]*reportingapi.Ref`, so I'm not clear on why we don't copy this from the reportControlMin to the convertedControl directly.

Reply:
reportControlMin.Refs was a different type, ESInSpecReportControlRef - which is why we had to marshal/unmarshal, but after taking a look i figured i can just make it the same type throughout and then just copy as you said 🎉

Reply:
> reportControlMin.Refs was a different type, ESInSpecReportControlRef 

Interesting! My editor lied to me! I shall forgive it, but just this once.

Thanks for fixing this up nonetheless :D

Reply:
so i ended up reverting the change where i made the types the same; see commit msg: 
```
there was some discussion on the pr about this. making the ref
type consistent means putting the api defined type in the es structs
file, which is bad practice b/c we're then bound to the api definition of
the struct when dealing with elasticsearch. so reverting the change!
```

Reply:
That's cool.  Note that we could still avoid round-tripping through JSON with something like (warning: code written in comment window):

```golang
convertedControl.Refs = make([]*reportingapi.Ref, len(reportControlMin.Refs))
for i, ref := range reportControlMin.Refs {
	convertedControl.Refs[i] = &reportingapi.Ref{
		Ref: ref.Ref,
		Url: ref.Url,
	}
}
```

My _guess_ is this is a bit faster and has the benefit of not having any errors to handle.  However, I'm also fine with leaving this for some future cleanup PR.

Reply:
that does seem safer than the json round tripping -- ill make the change. thank you!!